Python/import.c

```diff
@@ -2784,6 +2785,18 @@ _imp_is_lazy_import_impl(PyObject *module, PyObject *dict, PyObject *key)
     Py_RETURN_FALSE;
 }
 
+static PyObject *
+_imp_set_lazy_imports_impl(PyObject *module)
+{
+    // enable lazy imports
+    int lazy_imports_status = PyImport_EnableLazyImports();
+
+    if (lazy_imports_status == 1) {
+        Py_RETURN_TRUE;
+    }
+    Py_RETURN_FALSE;
```

Comment:
I tried, but it seems we cannot use `void` in a library method. It will cause this error,
`Fatal Python error: _Py_CheckFunctionResult: a function returned NULL without setting an exception`.

Reply:
ah, good point, it should return `PyObject *`, but we can simply `Py_RETURN_NONE` when we don't have a useful return value